### PR TITLE
Release 1.3.4

### PR DESCRIPTION
**Pull request by CyberDem0n**
Fix documentation and update release notes

### CODE REVIEW

File: docs/releases.rst [hunk 1]
```diff
@@ -3,6 +3,70 @@
...
+- create empty ``pg_ident.conf`` file if it is missing after restoring from the backup (Alexander)
+- open access in ``pg_hba.conf`` to all databases, not only postgres (Franco Bellagamba)
+
+
+Version 1.3.2
+-------------
+
+**Bugfix**
+
+- patronictl edit-config didn't worked with ZooKeeper (Alexander Kukushkin)
```
Comment (alexeyklyukin) [inline docs/releases.rst]: didn't work

File: docs/releases.rst [hunk 2]
```diff
@@ -3,6 +3,70 @@
...
+  new python-consul module has changed signature of some methods
+
+- "Could not take out TTL lock" message was never logged (Alexander)
+
+  Not a critical bug, but lack of proper logging complicates investigation in case of problems.
+
+
+**Quote synchronous_standby_names using quote_ident**
+
+- When writing ``synchronous_standby_names`` into the ``postgresql.conf`` it's value must be quoted (Alexander)
```
Comment (alexeyklyukin) [inline docs/releases.rst]: its value

File: docs/releases.rst [hunk 3]
```diff
@@ -3,6 +3,70 @@
...
+- Set correct postgres state in pause mode
+- Do not try to run queries from API if postgres is stopped
+
+
+Version 1.3.3
+-------------
+
+**Bugfixes**
+
+- short after promote synchronous replication was disabled even is synchronous_mode_strict is set (Alexander Kukushkin)
```
Comment (alexeyklyukin) [inline docs/releases.rst]: synchronous replication was disabled shortly after promotion even when synchronous_mode_strict was turned on.

**Comment by alexeyklyukin** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**